packages/hello-gsm-admin/src/PageContainer/MainPage/style.ts

```diff
@@ -94,16 +94,11 @@ export const Print = styled.a`
 `;
 
 export const ContentList = styled.div`
-  width: 100%;
   height: 600px;
   overflow-y: scroll;
+
   &::-webkit-scrollbar {
-    width: 7px;
-  }
-  &::-webkit-scrollbar-thumb {
-    background: #ffffff;
-    box-shadow: -2px 4px 8px rgba(0, 0, 0, 0.28);
-    border-radius: 10px;
+    width: 0px;
```

Comment:
이 코드의 목적이 뭔가요? 스크롤바를 지우는 건가요?

Reply:
넵 스크롤바 때문에 ContentBox 크기가 줄어들어서 스크롤바를 지우기 위해 작성했습니다

Reply:
```suggestion
  -ms-overflow-style: none;
  &::-webkit-scrollbar {
    display: none;
  }
```
width를 0으로 줄이는 것보단 해당 코드처럼 스크롤 바를 아예 없애는게 맞는 것 같습니다!

Reply:
넵 반영하겠습니다! 감사합니다